internal/core/src/index/InvertedIndexTantivy.cpp

```diff
@@ -188,30 +188,32 @@ InvertedIndexTantivy<T>::Load(milvus::tracer::TraceContext ctx,
                                      }),
                       files_value.end());
 
-    auto index_valid_data_file =
-        mem_file_manager_->GetRemoteIndexObjectPrefix() +
-        std::string("/index_null_offset");
-    auto it = std::find(
-        files_value.begin(), files_value.end(), index_valid_data_file);
-    if (it != files_value.end()) {
-        files_value.erase(it);
-        std::vector<std::string> file;
-        file.push_back(index_valid_data_file);
-        auto index_datas = mem_file_manager_->LoadIndexToMemory(file);
-        AssembleIndexDatas(index_datas);
-        BinarySet binary_set;
-        for (auto& [key, data] : index_datas) {
-            auto size = data->DataSize();
-            auto deleter = [&](uint8_t*) {};  // avoid repeated deconstruction
-            auto buf = std::shared_ptr<uint8_t[]>(
-                (uint8_t*)const_cast<void*>(data->Data()), deleter);
-            binary_set.Append(key, buf, size);
+    for (auto idx = 0; idx < files_value.size(); idx++) {
```

Comment:
this new code looks not simple and clean as original

Reply:
Is the modified logic the same as before?